src/obis-observer/commands/uplink/GetMeterInfoResponse.ts

```diff
@@ -82,8 +98,12 @@ class GetMeterInfoResponse extends Command {
         const buffer = new CommandBinaryBuffer(data);
 
         const requestId = buffer.getUint8();
```

Comment:
remove one empty line before and add one after

Reply:
fixed